### ISSUE TITLE
Slim-walkthrough doesn't classify flowers very well

### ISSUE THREAD

**Issue by freeman-g** [tensorflow/models#789]
## slim
I ran the slim-walkthrough and the last section:

```
Apply fine tuned model to some images.
```

Did not classify very well.  

I ran a batch of 6 images and these are the results.  

```
Ground Truth: [daisy], Prediction [tulips]
Ground Truth: [sunflowers], Prediction [tulips]
Ground Truth: [dandelion], Prediction [tulips]
Ground Truth: [dandelion], Prediction [sunflowers]
Ground Truth: [daisy], Prediction [daisy]
Ground Truth: [tulips], Prediction [daisy]
```

Is this expected or did I do something wrong?

This is the output from the previous section `We will fine tune the inception model on the Flowers dataset.`

```
WARNING:tensorflow:From <ipython-input-17-e0d720097de2>:52 in <module>.: scalar_summary (from tensorflow.python.ops.logging_ops) is deprecated and will be removed after 2016-11-30.
Instructions for updating:
Please switch to tf.summary.scalar. Note that tf.summary.scalar uses the node name instead of the tag. This means that TensorFlow will automatically de-duplicate summary names based on the scope they are created in. Also, passing a tensor or list of tags to a scalar summary op is no longer supported.
WARNING:tensorflow:From /usr/local/lib/python2.7/dist-packages/tensorflow/python/training/supervisor.py:344 in __init__.: __init__ (from tensorflow.python.training.summary_io) is deprecated and will be removed after 2016-11-30.
Instructions for updating:
Please switch to tf.summary.FileWriter. The interface and behavior is the same; this is just a rename.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global step 4: loss = 4.3324 (11.73 sec/step)
INFO:tensorflow:Stopping Training.
INFO:tensorflow:Finished training! Saving model to disk.
Finished training. Last batch loss 4.332441WARNING:tensorflow:From <ipython-input-17-e0d720097de2>:52 in <module>.: scalar_summary (from tensorflow.python.ops.logging_ops) is deprecated and will be removed after 2016-11-30.
Instructions for updating:
Please switch to tf.summary.scalar. Note that tf.summary.scalar uses the node name instead of the tag. This means that TensorFlow will automatically de-duplicate summary names based on the scope they are created in. Also, passing a tensor or list of tags to a scalar summary op is no longer supported.
WARNING:tensorflow:From /usr/local/lib/python2.7/dist-packages/tensorflow/python/training/supervisor.py:344 in __init__.: __init__ (from tensorflow.python.training.summary_io) is deprecated and will be removed after 2016-11-30.
Instructions for updating:
Please switch to tf.summary.FileWriter. The interface and behavior is the same; this is just a rename.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global step 4: loss = 4.3324 (11.73 sec/step)
INFO:tensorflow:Stopping Training.
INFO:tensorflow:Finished training! Saving model to disk.
Finished training. Last batch loss 4.332441
```

**Comment by ogail** [comment]
I think this is expected, the fine-tuning is done for 4 steps only which is very small. If you wanted to get better accuracy mimic what is in the [inception v3 fine-tuning script](https://github.com/tensorflow/models/blob/master/slim/scripts/finetune_inception_v3_on_flowers.sh) mainly by fine-tuning last layer with big learning rate and all network will smaller one for few thousands of steps. The accuracy I got out of this was ~90% accurate

**Comment by mingxingtan** [comment]
Close due to inactivity for a long time.
